pkg/docker/config/config_test.go

```diff
@@ -675,3 +676,170 @@ func TestAuthKeysForRef(t *testing.T) {
 		require.Equal(t, tc.expected, result, tc.name)
 	}
 }
+
+func TestSetCredentials(t *testing.T) {
+	for _, tc := range []struct {
+		registries []string
+		assert     func(dockerConfigFile)
+	}{
+		{
+			registries: []string{"quay.io"},
+			assert: func(auth dockerConfigFile) {
+				require.Len(t, auth.AuthConfigs, 1)
+				require.NotEmpty(t, auth.AuthConfigs["quay.io"].Auth)
```

Comment:
Non-blocking, throughout: `require.` for conditions that break tests, e.g. a failure setting up the environment, or an unexpected error we don’t get a value necessary for other checks; `assert.` for conditions that are a test failure but still allow continuing tests.

OTOH, sure, a test failure is a test failure, and needs to be fixed much more urgently than tinkering with how the test failure looks :)

Reply:
Got it, fixed as suggested :+1: